cogs/Staff.py

```diff
@@ -76,14 +76,14 @@ async def on_ready(self):
     async def ban(
         self,
         inter: ApplicationCommandInteraction,
-        member: disnake.Member,
+        user: disnake.User,
```

Comment:
Don't have a big enough problem with this to change but
Disnake.Member Represents a Discord member to a Guild.

User just represents a discord user

Reply:
now you can ban people wich are not in the server atm